app/scripts/config.js

```diff
@@ -7,8 +7,8 @@ define([
   'use strict';
 
   var config = {
-    oauth: {
-      authEndpoint: '{%= config.get("server.oauth.authEndpoint") %}'
+    embedly: {
+      apiKey: '{%= config.get("embedly.apiKey") %}'
```

Comment:
It's how they suggest doing it... We can restrict to certain IPs and referrers in account settings. We could also add our own endpoint that proxies back to theirs... so many proxies!